### ISSUE TITLE
Speech Broken for Menus in 4.5.1

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#27627]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

When using arrows to navigate through menus the first item on each list including sub branches for arrowing right announces
“MuseScore Studio Window, MuseScore Studio, MuseScore Studio, MuseScore 4 Window, MuseScore 4 Window.”
It also does not seem possible to arrow right/left across all the menu items anymore, I got slightly different results each time.

STEPS:
1.  Press Alt F to open the File menu and then right arrow.
2. You hear the above repetition of "MuseScore Studio, etc"
3. You should hear "Edit Menu" or something similar.
1.  Press Alt A to enter the Add menu
2. Press Up arrow twice and you will hear Text.
3. Press Right arrow.
4. Again you hear the "MuseScore Studio, etc" instead of Title.
1.  Press Alt F to open the File Menu,
2. Press right arrow several times,
3. Navigation seems to stop after the Add menu
4. Try Alt H for Help and arrow left, navigation stops there somewhere also and won't continue all the way through

### Supporting files, videos and screenshots

N/A

### What is the latest version of MuseScore Studio where this issue is present?

4.5.1

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by MarcSabatella** [comment]
As I understand it, the main regression in 4.5.1 is that upon pressing Alt+F, instead of reading something to tell you the File menu has been opened and that New is selected, both Narrator and NVDA screen reader simply reads "MuseScore 4 Studio" (NVDA reads it twice).  This is a regression versus 4.4.4.  The workaround in 4.5.1 is, press Down then Up to hear that first element read, but it still doesn't read the name of the menu.

However, both Narrator and NVDA seem to do fine with the current 4.5.2 nightly build.

With Orca on Linux, 4.5.1 works fine.  But in at least one recent 4.5.2 build, *nothing* was read upon pressing Alt+F.  Only after pressing Down then Up did I hear "New".  But now, it's working in last night's build again, so I don't know what to think.

Anyhow, for me, bottom line is that as far as I can tell, all is well in 4.5.2, but it is a little mysterious.

And yes, when navigating between top level menus with Left/Right and you hit a menu with everything disabled, navigation gets hung up, but I don't think this is new.  Press Esc seems to get past it.

**Comment by jmwmusic** [comment]
Hi, Previously if pressing Alt F then yes the word New as the top item of the list was announced by NVDA.  But if I then arrowed left or right when it moved onto a new menu it announced the menu name, with some extra verbosity, something like menu application Edit, application Add or similar, but it did say what the menu was.  although the movement along with arrows was a bit clunky and sometimes took a second press for what seemed like a blank space it was I believe possible to arrow along the top through all the menus to discover them previously.  This is pretty important as if you don't know what is there it can be hard to remember or locate a shortcut if you cant browse through.  currently i can't arrow across all the menus without getting hung up somewhere.

**Comment by jmwmusic** [comment]
This seems to all work fine in 4.5.2
